pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -148,7 +148,8 @@ def run_pair(A_files, B_files, caliBrate, spectrograph, det, parset, show=False,
                                       vel_corr=None,
                                       vel_type=parset['calibrations']['wavelengths']['refframe'],
                                       tilts=tilts,
-                                      slits=copy.deepcopy(caliBrate.slits))
+                                      slits=copy.deepcopy(caliBrate.slits),
+                                      maskdef_designtab=None)
```

Comment:
Can you just make the default to these instantiations maskdef_designtab=None and hence just omit this optional variable in both cases here?